template-coq/src/quoter.ml

```diff
@@ -404,13 +404,15 @@ struct
         | Undef _ -> None
         | Primitive _ -> CErrors.user_err Pp.(str "Primitives are unsupported by TemplateCoq")
 	      | Def cs -> Some (Mod_subst.force_constr cs)
-	      | OpaqueDef lc -> 
+	      | OpaqueDef lc ->
+          if bypass then
```

Comment:
The name `bypass` makes this test appear backwards. Maybe call it `opaque`

Reply:
@gmalecha Do you mean `opaque=true` means do not quote opaque? I followed the same pattern as for `tmQuoteConstant`, it uses the `bypass` argument for that purpose.

Reply:
I think `bypass` sounds more descriptive than `opaque`.